samples/drivers/uart/stm32/single_wire/README.rst

```diff
@@ -0,0 +1,33 @@
+.. _single_wire:
+
+Single Wire UART
```

Comment:
This will show up directly here: https://docs.zephyrproject.org/latest/samples/boards/boards.html.
Please rename as `STM32 Single Wire UART`

Reply:
Done